lte/gateway/python/magma/mobilityd/subscriberdb_client.py

```diff
@@ -21,6 +21,7 @@
 class SubscriberDbClient:
     def __init__(self, subscriberdb_rpc_stub):
         self.subscriber_client = subscriberdb_rpc_stub
+        self.apn_vlan_map = {}
```

Comment:
why bother caching these in memory? how hot is this codepath going to be and will RPC overhead be significant enough? you're going to have to do some cache invalidation if you want to do this because these mappings could change.

Reply:
I can remove cache, we can introduce it if needed.
I thought APN reconf would trigger service restart which invalidates cache.